pyproject.toml

```diff
@@ -31,7 +31,7 @@ dynamic = ["version"]
 test = ["pytest", "pyshacl"]
 code_style = ["isort", "black", "flake8"]
 graph_generation = ["pygraphviz", "networkx"]
-development = ["black", "flake8", "isort", "networkx", "pytest"]
+development = ["black", "flake8", "isort", "networkx", "pytest", "pyshacl"]
```

Comment:
This looks like an oversight; I don't see how pyshacl is relevant for this PR.

Reply:
Good catch! If you like, you can open another PR to add this. Else, I'll do it. :)